promql/engine_test.go

```diff
@@ -198,7 +199,7 @@ func TestQueryError(t *testing.T) {
 }
 
 func TestEngineShutdown(t *testing.T) {
-	engine := NewEngine(nil, nil, 10, 10*time.Second)
+	engine := NewEngine(nil, prometheus.DefaultRegisterer, 10, 10*time.Second)
```

Comment:
Why did you make this change? This could interfere with other tests.

Reply:
My bad. I forgot to remove this. Thanks!